joomla/joomla-cms
# Fix save user without selecting user groups

Pull Request for Issue #13653.

### Summary of Changes
This PR prevents add/edit an account without selecting a user groups in the backend of your site. It also fixes a notice Undefined index: groups... when administrator update his own account from backend

### Testing Instructions
Test case 1: Login to administrator are of your site using super admin account, try to add/edit an user

1. Before patch, you can add/edit an account without selecting a user group
2. After patch, if you try to add/edit an account without selecting at least one user groups, error will be thrown and account won't be saved

Test case 2: Login to administrator area of your site using an administrator account, edit your account, make sure it is saved properly

### Documentation Changes Required

None
